ansible/roles/aws_ecs/tasks/main.yml

```diff
@@ -1,5 +1,5 @@
 - include_vars: aws_ecs.yml
-- file: name=/var/run/ecs_agent mode=0750 owner=root group=root state=directory
+- file: name=/var/lib/ecs_agent mode=0750 owner=root group=root state=directory
```

Comment:
This change should be considered.  /var/run gets wiped out on reboot, and the ECS agent registers itself as a new instance every time a minion is restarted.